data/events/scripts/monster.lua

```diff
@@ -43,18 +43,18 @@ function Monster:onDropLoot(corpse)
 		end
 
 		-- Vip system
-		local vipPercentLoot = 0
-		if (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED)
-			and configManager.getBoolean(configKeys.VIP_SYSTEM_LOOT_ENABLED)
-			and player and player:isVip()) then
-			vipPercentLoot = (configManager.getNumber(configKeys.VIP_SYSTEM_LOOT_PERCENT) / 100)
+		local vipLootPercent = 0
+		if configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) then
+			local percent = configManager.getNumber(configKeys.VIP_SYSTEM_EXP_PERCENT)
+			if (percent > 0 and player and player:isVip()) then
```

Comment:
```suggestion
if percent > 0 and player and player:isVip() then
```
Corrigir formatação.

Reply:
não é uma correção. e a indentação está errada tbm (ela tem que seguir toda a organização do código anterior).